tests/unit/service/test_database.py

```diff
@@ -17,20 +18,23 @@ def test_database_create(
     settings: Settings,
     account_id_callback: Callable,
     account_id_url: str,
-    databases_callback: Callable,
+    create_databases_callback: Callable,
     databases_url: str,
     db_name: str,
+    db_description: str,
 ):
     httpx_mock.add_callback(auth_callback, url=auth_url)
     httpx_mock.add_callback(provider_callback, url=provider_url)
     httpx_mock.add_callback(account_id_callback, url=account_id_url)
     httpx_mock.add_callback(auth_callback, url=auth_url)
     httpx_mock.add_callback(region_callback, url=region_url)
-    httpx_mock.add_callback(databases_callback, url=databases_url, method="POST")
+    httpx_mock.add_callback(create_databases_callback, url=databases_url, method="POST")
 
     manager = ResourceManager(settings=settings)
-    database = manager.databases.create(name=db_name)
+    database = manager.databases.create(name=db_name, description=db_description)
+
     assert database.name == db_name
+    assert database.description == db_description
```

Comment:
Should this, similarly to L64, use mock_database and `mock_database.description`?

Reply:
No, here we create database with name `db_name`, and this name should be returned by the function. In other test we request an "existing" database, and add a mock function that returns `mock_database.name`